libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnector.java

```diff
@@ -133,6 +146,20 @@ public InputStream unsafeGetAttachmentStream(AttachmentContent attachment) throw
         return in;
     }
 
+    private String printAcceptedFileName(String name, int count) {
+        String acceptedFileName;
```

Comment:
Why not return directly instead of this variable?

Reply:
changed to direct returns.